.github/CODEOWNERS

```diff
@@ -0,0 +1,243 @@
+/helmfile.d/charts/calico-felix-metrics/ @elastisys/goto-kubernetes
+/helmfile.d/charts/cluster-admin-rbac/ @elastisys/goto-security
+/helmfile.d/charts/external-dns-endpoints/ @elastisys/goto-ingress-auth
+/helmfile.d/charts/external-dns-secrets/ @elastisys/goto-ingress-auth
+/helmfile.d/charts/falco-psp-rbac/ @elastisys/goto-security
+/helmfile.d/charts/file-copier/ @elastisys/goto-ingress-auth
+/helmfile.d/charts/gatekeeper/ @elastisys/goto-security
+# TODO Refine per dashboard later
+/helmfile.d/charts/grafana-dashboards/ @elastisys/goto-monitoring-stack
+/helmfile.d/charts/grafana-label-enforcer/ @elastisys/goto-monitoring-stack
+/helmfile.d/charts/harbor/ @elastisys/goto-container-registry
+/helmfile.d/charts/hnc/ @elastisys/goto-security
+/helmfile.d/charts/ingress-nginx-probe-ingress/ @elastisys/goto-ingress-auth
+/helmfile.d/charts/issuers/ @elastisys/goto-ingress-auth
+/helmfile.d/charts/kube-state-metrics-extra-resource-metrics/ @elastisys/goto-monitoring-stack
+/helmfile.d/charts/kubeapi-metrics @elastisys/goto-monitoring-stack
+/helmfile.d/charts/kured-secret/ @elastisys/goto-security
+/helmfile.d/charts/log-manager/ @elastisys/goto-logging-stack
+/helmfile.d/charts/namespaces/ @elastisys/goto-kubernetes
+/helmfile.d/charts/networkpolicy/ @elastisys/goto-security
+/helmfile.d/charts/node-local-dns/ @elastisys/goto-kubernetes
+/helmfile.d/charts/opensearch/ @elastisys/goto-logging-stack
+/helmfile.d/charts/openstack-monitoring/ @elastisys/goto-kubernetes
+/helmfile.d/charts/prometheus-alerts/ @elastisys/goto-monitoring-stack
+/helmfile.d/charts/prometheus-servicemonitor/ @elastisys/goto-monitoring-stack
+/helmfile.d/charts/rclone/ @elastisys/goto-security
+/helmfile.d/charts/s3-exporter/ @elastisys/goto-monitoring-stack
+/helmfile.d/charts/tekton-pipelines/ @elastisys/goto-continuous-delivery
+/helmfile.d/charts/thanos/ @elastisys/goto-monitoring-stack
+/helmfile.d/charts/user-alertmanager/ @elastisys/goto-monitoring-stack
+/helmfile.d/charts/user-crds/ @elastisys/goto-security
+/helmfile.d/charts/user-rbac/ @elastisys/goto-security
+/helmfile.d/charts/README.md @elastisys/goto-scripts
+
+#
+# Skipped files in helmfile.d/charts/
+# .gitignore
+# public.gpg
+#
+
+#
+# helmfile.d/hooks/
+#
+
+/helmfile.d/hooks/dev-rbac/ @elastisys/goto-security
+/helmfile.d/hooks/fluentd/ @elastisys/goto-logging-stack
+/helmfile.d/hooks/create-from-manifest.sh @elastisys/goto-scripts
+
+#
+# helmfile.d/stacks/
+#
+
+/helmfile.d/stacks/calico.yaml @elastisys/goto-kubernetes
+/helmfile.d/stacks/cert-manager.yaml @elastisys/goto-ingress-auth
+/helmfile.d/stacks/dex.yaml @elastisys/goto-ingress-auth
+/helmfile.d/stacks/external-dns.yaml @elastisys/goto-ingress-auth
+/helmfile.d/stacks/falco.yaml @elastisys/goto-security
+/helmfile.d/stacks/fluentd.yaml @elastisys/goto-logging-stack
+/helmfile.d/stacks/gatekeeper.yaml @elastisys/goto-security
+/helmfile.d/stacks/hnc.yaml @elastisys/goto-security
+/helmfile.d/stacks/harbor.yaml @elastisys/goto-security
```

Comment:
Should this not be @elastisys/goto-container-registry ?

Reply:
Yep, copy paste error. Fixed now